project/AppDependencies.scala

```diff
@@ -6,21 +6,22 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "4.1.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.51.0-play-27",
-    "uk.gov.hmrc" %% "play-frontend-govuk" % "0.65.0-play-27",
-    "uk.gov.hmrc" %% "simple-reactivemongo" % "8.0.0-play-27"
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.4.0",
```

Comment:
Can these go to the latest ones on the Tax Platform Catalogue?

Reply:
done